theano/gof/tests/test_vm.py

```diff
@@ -441,6 +441,8 @@ def check_storage(storage_map):
 
 
 def test_no_recycling():
+    if theano.config.cxx == '':
+        raise SkipTest('need c++')
     x = theano.tensor.vector()
     for lnk in [vm.VM_Linker(use_cloop=True),
```

Comment:
This is a test that the generated C code is fine with the no_recycling change that @nouiz did.

I don't see the point of testing this with no C++ compiler.